lib/ansible/modules/windows/win_certificate_store.py

```diff
@@ -91,7 +91,7 @@
   key_storage:
     description:
     - Specifies where Windows will store the private key when it is imported.
-    - When set to C(default), the default option as set by Windows is used.
+    - When set to C(default), the default option as set by Windows is used- typically user.
```

Comment:
probably best to turn the `-` into `,` and enclose user with `C(user)`.

Reply:
Done.